map_it_test.go

```diff
@@ -1071,6 +1063,145 @@ func TestMap_ExecuteOnEntries(t *testing.T) {
 	})
 }
 
+func TestMap_AddInterceptor(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		ctx := context.Background()
+		prefix := "My Prefix"
+		id, err := m.AddInterceptor(ctx, &MapGetInterceptor{Prefix: prefix})
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func() {
+			if _, err := m.RemoveInterceptor(ctx, id); err != nil {
+				panic(err)
```

Comment:
Why do we panic instead of t.Fatal() ?

Reply:
It wouldn't matter, but switched to `t.Fatal` to be more consistent.